app/assets/stylesheets/generic/nav_sidebar.scss

```diff
@@ -39,7 +39,7 @@
 
 .nav-sidebar li {
   &.active a {
-    color: #333;
+    color: $text-color;
```

Comment:
Properties should be ordered background, border, border-left, border-right, color, font-weight